### PR TITLE
Initial version of dojot's cron microservice

### PR DESCRIPTION
**Pull request by rascaraficci**
This microservice allows to schedule jobs that emits
events to other services through HTTP requests or Kafka events.

The current version doesn't perist the jobs' configuration, so
if the service is restarted, all jobs will be missed.

### CODE REVIEW

File: docs/api.apib [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+
+            {
+                [
+                    {
+                        "jobId":    "e909e5c5-c0a7-4d91-a60a-5dbfe17fcdad",
+                        "spec":     {
+
+                                        "time":        "*/5 * * * *",
+                                        "timezone":    "America/Sao_Paulo",
+                                        "name":        "Keep alive\",
```
Comment (raulnegreiros) [inline docs/api.apib]: `"Keep alive\",` I think this backslash is unnecessary.

File: cron.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+                let [_tenant, jobId] = key.split(':');
+                if(_tenant === tenant) {
+                    jobs.push({jobId: jobId, spec: value.spec});
+                }
+            }
+            resolve(jobs);
+        });
+    }
+
+/*
```
Comment (raulnegreiros) [inline cron.js]: some reason to this method be commented?

**Review by raulnegreiros — COMMENTED** [review]
There is some reason for the code related to jscode be commented?

File: docs/api.apib (hunk 1 above)
Comment (rascaraficci) [reply docs/api.apib]: You're right!

File: cron.js (hunk 2 above)
Comment (rascaraficci) [reply cron.js]: Just because I haven't had time to validate it.

**Review by raulnegreiros — APPROVED** [review]

**Pull request merged**